packages/caip/src/adapters/osmosis/utils.ts

```diff
@@ -39,11 +39,11 @@ export const parseOsmosisData = (data: OsmosisCoin[]) => {
 
     if (isNativeAsset) {
       // TODO(ryankk): remove `toString` when AssetReferences are changed to strings
-      assetReference = isOsmo ? AssetReference.Osmosis.toString() : denom
-      assetNamespace = isOsmo ? AssetNamespace.Slip44 : AssetNamespace.NATIVE
+      assetReference = isOsmo ? ASSET_REFERENCE.Osmosis.toString() : denom
```

Comment:
Can remove `toString()` and comment above

Reply:
Sounds good - removed in https://github.com/shapeshift/lib/pull/656/commits/f2b3ea985a4f48f4d3e7ba2e14bd3491cfecf48c